bindings/java/src/org/sleuthkit/datamodel/BlackboardArtifact.java

```diff
@@ -218,7 +218,9 @@ public int hashCode() {
 		TSK_REMOTE_DRIVE(37, "TSK_REMOTE_DRIVE", //NON-NLS
 				bundle.getString("BlackboardArtifact.tskRemoteDrive.text")),
 		TSK_FACE_DETECTED(38, "TSK_FACE_DETECTED", //NON-NLS
-				bundle.getString("BlackboardArtifact.tskFaceDetected.text"));
+				bundle.getString("BlackboardArtifact.tskFaceDetected.text")),
+		TSK_ACCOUNT(39, "TSK_ACCOUNT", //NON-NLS
```

Comment:
This would probably collide with what intended to be changed here: https://github.com/sleuthkit/sleuthkit/blob/develop/API-CHANGES.txt#L2